cypress/integration/product-page.js

```diff
@@ -0,0 +1,52 @@
+/* eslint no-undef: 0 */
+describe('product page', () => {
+  beforeEach(() => {
+    cy.visit('/p/6dbfdfd5-2e34-4dd6-ac18-c0cc98b84d78/poloralphlauren-sweater-A40S4603-pink').wait(2000);
+  });
+  it('should verify that all information are visible', () => {
+    cy.location('pathname').should('include', 'poloralphlauren-sweater');
+    cy.get('.sf-heading__title').contains('Sweater Polo Ralph Lauren pink');
+    cy.get('.sf-price__value').contains('$160.00');
+    cy.get('.product__gallery > .sf-image > img').should(
+      'have.attr',
+      'src',
+      'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/079922_1_large.jpg'
+    );
+    cy.get('[data-cy=product-color_update]').should('have.css', 'background-color', 'rgb(255, 192, 203)');
+    cy.get('[data-cy=product-select_size]').first().contains('XXS');
+    cy.get('.sf-select__options').children().should('have.length', 8);
+    cy.wrap(['XXS', 'XS', 'S', 'M', 'L', 'XL', 'XXL', 'XXXL']).its(1).should('eq', 'XS');
```

Comment:
here you are also testing array which will not change, I think it would be better to get elements and check inner text for each of them